src/intel_driver.c

```diff
@@ -56,6 +56,51 @@ uint32_t g_intel_debug_option_flags = 0;
 #define LOCAL_I915_PARAM_EU_TOTAL 34
 #endif
 
+bool intel_batchbuffer_configure_watchdog(int fd, unsigned int gem_ctx_id, int flag)
+{
+    unsigned threshold[5];
```

Comment:
From the other place this is used I suppose this should be u32. I suppose we will have a look at the uAPI details when the i915 part gets posted.